src/main/java/com/philderbeast/prisonpicks/Xpick.java

```diff
@@ -61,6 +65,11 @@ public void breakBlock(BlockBreakEvent event)
                             && (block.getBlock().getType() != Material.BEDROCK)
                             && (block.getBlock().getType() != Material.AIR))
                         {
+                            if (block.getWorld().getBlockAt(block).getType() == Material.EMERALD_ORE) {
```

Comment:
why are you trying to send messages in the middle of finding the blocks to break?

Reply:
if you break a emerald with the xpick it will send the message ~ working

Reply:
this is the worst possible place to put it, it will end up getting spammed if there is more then one in the loop and slow the overall loop down.

if this is really needed make a flag and send the message outside the loop.